aten/src/ATen/native/README.md

```diff
@@ -126,6 +112,37 @@ Here are the supported default values:
   a `Tensor` argument to accept undefined tensors).
 * `nullptr` for pointer types (e.g., `Generator*`)
 
+**Returns.** The following are permissible on Return:
+
+Non-tuple return:
+```
+ReturnType [retarg0]
+```
+
+Tuple return:
+```
+(ReturnType [retarg0], ReturnType [retarg1], ...)
+```
+
+The following are permissible on ReturnType:
+- `Tensor` and `TensorList`, which translate into the C++ types `Tensor` and `std::vector<Tensor>`,
+  respectively (unless the operation is in-place, in which case the return type
+  is `Tensor&`.
```

Comment:
Oh nooooo, can we not do this please. The fact that we return a mutable tensor ref is a mistake, let's not enshrine it in the syntax of native functions. (Didn't we have a proposal for syntax for mutation?)

Reply:
This isn't relevant; there are a couple of things going on here:
1) This section is all literally moved, besides the part that explains you can now give return arguments optional names.  I just moved it below all the argument examples for clarity.
2) The part you are objecting to (which again, is literally moved), is referring to the C++ return type, not the native function return type.